src/components/plots/ProductConcentrationPlot.tsx

```diff
@@ -109,6 +98,25 @@ const ProductConcentrationPlot: React.FC<ProductConcentrationPlotProps> = ({
         ...BASE_PLOT_LAYOUT,
         width: width,
         height: Math.max(130, height),
+        legend: {
+            title: {
+                text: "Initial [B]",
+                // TODO: uncomment out next line once PR is merged to update @types/plotly.js
```

Comment:
I'm not sure I understand why this cant be part of this PR...

Reply:
it's a PR to the open source repo that we install